client/src/components/organisms/AdminRequestGroupBrowser.tsx

```diff
@@ -171,7 +170,7 @@ const AdminRequestGroupBrowser: FunctionComponent = () => {
                     {showDeleteGroupModal && (
                         <DeleteRequestGroupDialog
                             requestGroupName={requestGroup.name}
-                            numRequests={requestGroup.countOpenRequests ?? 0}
+                            numRequests={numRequests}
```

Comment:
See [Slack](https://uwblueprint.slack.com/archives/C01L4QRS456/p1630367864027700?thread_ts=1630365211.026800&cid=C01L4QRS456) for description of what error they're referring to